web/template/partial/course/manage/lecture-management-card.gohtml

```diff
@@ -197,7 +197,7 @@
                             <button x-text="lectureData.private?'Make public': 'Make private'"
                                     class="block w-full px-4 py-2 text-left text-sm"
                                     @click="toggleVisibility(); closeMoreDropdown()"
-                                    x-show="lectureData.isRecording"
+                                    x-show="lectureData.isRecording || lectureData.isLiveNow || !lectureData.isPast && !lectureData.isLiveNow && !lectureData.isRecording"
```

Comment:
I am a bit confused by this expression:
Would this expression not be always true?

- `lectureData.isLiveNow`
- `!lectureData.isLiveNow`

and 

- `lectureData.isRecording`
- `!lectureData.isRecording`